RecoLocalCalo/HcalRecAlgos/BuildFile.xml

```diff
@@ -1,19 +1,26 @@
 <use name="boost"/>
-<use name="eigen"/>
 <use name="clhep"/>
+<use name="cuda"/>
+<use name="eigen"/>
+<use name="rootminuit2"/>
+<use name="vdt_headers"/>
+<use name="CUDADataFormats/Common" />
+<use name="CUDADataFormats/HcalDigi"/>
+<use name="CUDADataFormats/HcalRecHitSoA" />
+<use name="CalibCalorimetry/HcalAlgos"/>
+<use name="CalibFormats/CaloObjects"/>
+<use name="CalibFormats/HcalObjects"/>
 <use name="DataFormats/HcalDigi"/>
 <use name="DataFormats/HcalRecHit"/>
 <use name="DataFormats/TrackReco"/>
-<use name="CalibFormats/HcalObjects"/>
-<use name="CalibFormats/CaloObjects"/>
-<use name="CalibCalorimetry/HcalAlgos"/>
 <use name="FWCore/Framework"/>
-<use name="FWCore/ParameterSet"/>
 <use name="FWCore/MessageLogger"/>
-<use name="CondFormats/DataRecord"/>
```

Comment:
This gets removed: let me just check if it is wanted to do so, or if it got simply forgotten while reordering